### DISCUSSION TITLE
Torrents marked as finished despite ratio group action to 'continue'

### DISCUSSION THREAD

**Discussion by Span24** [Novik/ruTorrent#2416] [General]
I have my ratio group actions set to continue seeding despite the ratio. Nonetheless, once a seeded torrent tops 1:1 its marked as finished and seeding stops. I'm running multiple instances of v3.10 and all are configured exactly the same. Two are seedboxes and one a local unraid docker container. Only one of them (the local instance) is having this 'problem'.

What could be the cause?

**Comment by stickz** [comment]
Please consider upgrading to ruTorrent v4.0-beta3. We no longer support v3.10. It's required to replace the entire installation and clear your web browser cache. If the problem still exists file an issue report. We don't like bugs like this on ruTorrent.
https://github.com/Novik/ruTorrent/releases/tag/v4.0-beta3
